gradle/verification-metadata.xml

```diff
@@ -3229,11 +2711,6 @@
             <sha256 value="e469bd39f936999f256002631003ff022a22951da9d5bd9789c7abfa9763a292" origin="Generated by Gradle"/>
          </artifact>
       </component>
-      <component group="org.bouncycastle" name="bctls-fips" version="1.0.9">
```

Comment:
This jar is needed, but only when `tests.fips.enabled` system property is set to `true`.

Reply:
Added back via 6010970617be517ec5dd32ab3bead392cbb0d2be